src/Dependencies/VisualStudio15/VisualStudio15.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```

Comment:
Please move this to Closed, so that we aren't adding all these binary files to the repository history for all eternity.